docs/manual-EN/1.overview/2.quick-start/2.FAQ.md

```diff
@@ -0,0 +1,148 @@
+
+1. First run `ifconfig` in container to check your container IP, here we assume your IP is 172.17.0.3.
+2. In directory `/usr/local/nebula/etc`, check the config locations of all the IP addresses with the command `grep "172.17.0.2" . -r`.
+3. Change all the IPs you find in step 2 to your container IP 172.17.0.3.
+4. Restart all the services.
+
+### Check Logs
+
+Logs are stored under `/usr/local/nebula/logs/` by default.
+
+Logs details please refer to [logs](docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/log.md).
+
+### Check Configs
+
+Configuration files are stored under `/usr/local/nebula/etc/` by default.
+
+### Check runtime Configs
+
+In Nebula console, run
+
+```ngql
+nebula> SHOW CONFIGS;
+```
+
+Configuration details please see [here](docs/manual-CN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md).
+
+### Connection Refused
+
+```txt
+E1121 04:49:34.563858   256 GraphClient.cpp:54] Thrift rpc call failed: AsyncSocketException: connect failed, type = Socket not open, errno = 111 (Connection refused): Connection refused
+```
+
+Check service status by
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service status all
+```
+
+### Process Crash
+
+1. Check disk space `df -h`.
+1. Check memory usage `free -h`.
+
+### `Could not create logging file:... Too many open files`
+
+1. Check your disk space `df -h`
+1. Check log directory `/usr/local/nebula/logs/`
+1. reset your max open files by `ulimit -n 65536`
+
+### Storaged Service Cannot Start Normally
+
+When the same host is used for single host or cluster test, the storaged service cannot start normally. The listening port of the storaged service is red in the console.
+
+Check the logs of the storaged service. If you find the "wrong cluster" error message, the possible cause is that the cluster id generated by **Nebula Graph** during the single host test and the cluster test are inconsistent. You need to delete the cluster.id file and the data directory and restart the service.
+
+## General Information
+
+General Information lists the conceptual questions about **Nebula Graph**.
+
+### Explanations on the Time Return in Queries
+
+```ngql
+nebula> GO FROM 101 OVER follow
+===============
+| follow._dst |
+===============
+| 100         |
+---------------
+| 102         |
+---------------
+| 125         |
+---------------
+Got 3 rows (Time spent: 7431/10406 us)
+```
+
+Taking the above query as an example, the first number is the time it takes the query engine to receive the data from the console and pass the data to the storage engine to perform a series of calculations on the entire link; the second number is the time spent from sending the request to receiving the response and outputting the results in the console.
```

Comment:
`Time spent: 7431/10406 us`, the first number (`7431`) is the time consuming of the database.  i.e., the sum of a) receiving the query from CLI, b)  asking and getting back results from the storage engine (storaged),  c)  finishing computation in graphd; 
the second number (10406) is the time consuming from CLI's view. i.e., the sum of the above time plus d) request back to the CLI, and e) displaying the results on the screen.